Prog/New_Project_Geocode.sas

```diff
@@ -0,0 +1,565 @@
+/**************************************************************************
+ Program:  New_Project_Geocode.sas
+ Library:  PresCat
+ Project:  NeighborhoodInfo DC
+ Author:   M. Cohen
+ Created:  08/18/2016
+ Version:  SAS 9.1
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Adds new projects into the project_geocode and building_geocode files.
+
+ Modifications:
+
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+
+
+%let MAX_PROJ_ADDRE = 3;   /** Maximum number of addresses to include in Proj_addre field in PresCat.Project_geo **/
+%let outlib = %mif_select( &_remote_batch_submit, PresCat, WORK );
```

Comment:
I don't think we need the outlib macro var. We'll handle through a full macro instead.